app/index.js

```diff
@@ -60,10 +60,15 @@ function ConnectDialog () {
   self.channelName = ko.observable('')
   self.joinOnly = ko.observable(false)
   self.visible = ko.observable(true)
+  self.loading = ko.observable(false)
```

Comment:
The current name `loading` might be confused with loading the page (at least that's what I though at first), `connecting` seems like a better name for this.